src/store/Store.js

```diff
@@ -1,9 +1,9 @@
 /* eslint no-console:0 */
 /* @flow */
-import type { Action, Dispatcher } from 'flux';
+import type {Action, Dispatcher} from 'flux';
 import type StoreFactory from './StoreFactory';
 
-import { isPayload } from '../dispatcher/DispatcherInterface.js';
+import {isPayload} from '../dispatcher/DispatcherInterface.js';
```

Comment:
At the moment general-store's prettier config is different from HubSpot's. Can you run format before you commit this (https://github.com/HubSpot/general-store/blob/master/package.json#L14).

Reply:
I'll probably move it over to use hubspot's config in the future, but for now...

Reply:
Yeah I can do that. I always forget I have format on save on.